packages/web/src/scss/components/Footer/index.html

```diff
@@ -585,6 +585,123 @@ <h3 class="typography-heading-xsmall-semibold mb-700">
 
       </div>
     </footer>
+  </div>
+</section>
+
+<section class="UNSTABLE_Section">
+  <h2 class="docs-Heading">Minimalistic</h2>
+  <div class="docs-Stack">
+    
+    <footer class="bg-secondary pt-900 pb-900">
+      <div class="Container">
+
+        <!-- Grid with product logo, secondary links and language switch -->
+        <div class="Grid Grid--cols-1 Grid--desktop--cols-12 Grid--alignmentYCenter Grid--desktop--alignmentXStretch Grid--alignmentXCenter" style="--grid-spacing-y: var(--spirit-space-1100)">
+        
+          <!-- Product logo -->
+          <div class="GridItem text-desktop-left" style="--grid-item-column-start-desktop: 1; --grid-item-column-end-desktop: 4;">
+            <a href="https://www.example.com">
+              <div class="ProductLogo">
+                <svg width="178" height="44" viewBox="0 0 178 44" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We have this logo as a Handlebars partial, can we use it here (and actually everywhere)?